### PR TITLE
fixing jquery

### PR DESCRIPTION
**Pull request by hbaptiste**
Fix vendors and jquery dependencies.

### CODE REVIEW

**Comment by ndufreche** [comment]
:+1:

File: dist/config.js [hunk 1]
```diff
@@ -1,5 +1,10 @@
 
 /* src/require.config.js */
+/* global $:false */
+/*globals $*/
```
Comment (mickaelandrieu) [inline dist/config.js]: space is missing here

File: dist/config.js [hunk 2]
```diff
@@ -18,39 +23,48 @@
...
+    "use strict";
+    if (map.name === "jquery.noconflict") {
+        require.undef(map.name);
+    }
+};
+
 require.config({
     baseUrl: 'resources/toolbar/',
     catchError: true,
+    waitSeconds: 15,
```
Comment (mickaelandrieu) [inline dist/config.js]: can you explain me why you need to set a timeout in an asynchronous system ?

File: dist/config.js (hunk 1 above)
Comment (ndufreche) [inline dist/config.js]: no => http://www.jslint.com/lint.html#global

File: dist/config.js (hunk 2 above)
Comment (ndufreche) [inline dist/config.js]: It the time to wait before raise an error if nothing happen.
This is basically a require.js config, if you want more explanations I encourage you to watch the [require.js doc](http://requirejs.org/docs/api.html#config)

Comment (mickaelandrieu) [inline dist/config.js]: thanks for the link @ndufreche

File: dist/config.js (hunk 1 above)
Comment (mickaelandrieu) [inline dist/config.js]: ``/*globals`` doesnt exist in the specification and the right syntax is:

``/*global $:false */``

According to the specification you give, a space is not missing but to be removed on l.3 and the instruction on l.4 is wrong.

Are we ok ?

Comment (ndufreche) [inline dist/config.js]: It's possible, but i will correct this later, i have some upgrade to do on this file.

**Pull request merged**